contentful/README.md

```diff
@@ -0,0 +1,63 @@
+## Overview
+
+[Contentful][1] is a content (articles, photos, and videos) management platform that allows businesses to create, manage, and deliver digital content across various channels like websites and mobile apps through its intuitive interface and robust APIs or SDKs.
+
+Integrate with Datadog to gain insights into Contentful activities related to content and other actions as part of your Contenful spaces and environments.
+
+## Setup
+
+Follow the instructions below to configure this integration for your Contentful space.
+
+### Configuration
+
+#### Webhook Configuration
+Configure the Datadog endpoint to forward Contentful events as logs to Datadog. See [Contentful Webhook overview][2] for more details.
+
+1. Select an existing API key or create a new one by clicking one of the buttons below:<!-- UI Component to be added by DataDog team -->
```

Comment:
I don't know what the comment here is referring to.

I think this should just have a link to the API keys page.

Reply:
This comment is kept here because someone from the UI Datadog team will build the UI tile for this integration and once it is developed we will remove this line from here.
Joel Calce will create a ticket for the same.

Reply:
Okay - please let me know when this is complete and I will do another review.

Reply:
Removed this line and replaced with link to the new tile.